datajunction-query/djqs/engine.py

```diff
@@ -94,6 +96,17 @@ def run_query(
             )
         )
         return run_duckdb_query(query, conn)
+    if engine.type == EngineType.SNOWFLAKE:
+        conn = snowflake.connector.connect(
+            user=engine.extra_params["user"],
+            account=engine.extra_params["account"],
+            password=os.getenv("SNOWSQL_PWD"),
+            database=engine.extra_params["database"],
+        )
```

Comment:
There are a number of [other parameters](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example) to this `connect(...)` function. I wonder if this would be better if we just use unpacking here to allow people to define any of them. We should still plug-in the password from this `SNOWSQL_PWD` environment, so I'm thinking something like:

```py
    if engine.type == EngineType.SNOWFLAKE:
        conn = snowflake.connector.connect(
            **extra_params,
            password=os.getenv("SNOWSQL_PWD"),
        )
```

Reply:
@samredai I think this is cleaner and more flexible, thanks for for the suggestion. I will make the changes.